packages/db/fauna/schema.graphql

```diff
@@ -45,12 +45,12 @@ type AuthToken {
   """
   Uploads made using this token.
   """
-  uploads: [Upload!]! @relation @index
+  uploads: [Upload!]! @relation
```

Comment:
`@index` with no name is a noop - this doesn't actually tell Fauna to create an index on this field, just allows you to name the index to use.

https://docs.fauna.com/fauna/current/api/graphql/directives/d_index